lib/go-tc/users.go

```diff
@@ -127,9 +127,9 @@ type UserToken struct {
 	Token string `json:"t"`
 }
 
-// commonUserFields is unexported, but its contents are still visible when it is embedded
+// CommonUserFields is unexported, but its contents are still visible when it is embedded
```

Comment:
> CommonUserFields is unexported

is no longer true

also, though, I'm curious why you exported it at all?

Reply:
Because the tests aren't in the same package.